miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/analytics/MiniAppAnalytics.kt

```diff
@@ -43,9 +43,34 @@ internal class MiniAppAnalytics(val rasProjectId: String) {
                 .put("mini_app_version_id", miniAppInfo.version.versionId)
         }
         params["cp"] = cp
-
         RatTracker.event(eType.value, params).track()
+        // Send to all the external acc/aid added by host app
+        listOfExternalConfig.map { config ->
+                params["acc"] = config.acc
+                params["aid"] = config.aid
+                RatTracker.event(eType.value, params).track()
+        }
     } catch (e: Exception) {
         Log.e("MiniAppAnalytics", e.message.orEmpty())
     }
+
+    /** Is add single analytic configuration. **/
+    fun addAnalyticsConfig(miniAppAnalyticsConfig: MiniAppAnalyticsConfig) {
+        listOfExternalConfig.add(miniAppAnalyticsConfig)
+    }
+
+    /** Is add multiple analytic configuration. **/
+    fun addAnalyticsConfig(miniAppAnalyticsConfigs: List<MiniAppAnalyticsConfig>) {
```

Comment:
I think we don't need to have two versions of the function. Maybe you could use `vararg` instead? `fun addAnalyticsConfig(vararg miniAppAnalyticsConfig: MiniAppAnalyticsConfig)`